File: go/docmost/models/pagination_options.go
package models

// PaginationOptions from OpenAPI.
type PaginationOptions struct {
	Limit *int `json:"limit,omitempty"`
	Cursor *string `json:"cursor,omitempty"`
	Query *string `json:"query,omitempty"`
	AdminView *bool `json:"adminView,omitempty"`
}

File: c#/src/Docmost.Sdk/Models/PaginationOptions.cs
namespace Docmost.Sdk.Models;

public class PaginationOptions
{
    [System.Text.Json.Serialization.JsonPropertyName("limit")]
    public int? Limit { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("cursor")]
    public string? Cursor { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("query")]
    public string? Query { get; set; }

    [System.Text.Json.Serialization.JsonPropertyName("adminView")]
    public bool? AdminView { get; set; }

}
